storage/named_query.go

```diff
@@ -0,0 +1,49 @@
+package storage
+
+type NamedQuery int
+
+
+
+const (
+	QueryByMissingLabel NamedQuery = iota
+	QueryByExistingLabel
+)
+
+var namedQueries =  map[NamedQuery]string {
+
+	QueryByMissingLabel : `
```

Comment:
why do you need column alias here?

Reply:
the query fails if the alias are removed